W-18595520-secure-config-update-4.7-kt (#3049)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/secure-configuration-properties.adoc

k/J1EHyxJagOcyQELoaqIDCzzKc1bLGJiHY2dv6jzuhtVpP1LTlgK3y8EH7OkK5iuuu8kik3P5dCDMixZXwn/EV6xbUwLFKnEb4x1pUX6aopLP/KULm0pQ==
----

[NOTE]

> Fully encrypted files are not supported as secure properties or YAML configuration files because the system requires access to unencrypted keys for secure property resolution.

Reviewer: ```suggestion
Fully encrypted files aren't supported as secure properties or YAML configuration files because the system requires access to unencrypted keys for secure property resolution.
```
